activerecord/lib/active_record/migration.rb

```diff
@@ -613,6 +613,10 @@ def migrate(direction)
       def disable_ddl_transaction!
         @disable_ddl_transaction = true
       end
+
+      def connection
+        ActiveRecord::Base.connection
```

Comment:
👎 -- I'm not sure that's the right public API. We want to make it as easy as possible for users to be able to overwrite and change the `connection`, and I'm not sure that sticking this in the constructor is the best place -- I don't think passing in any options through this constructor is public API for users...